code-challenges.rb

```diff
@@ -21,6 +33,14 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowels (string)
+p string.delete "AEIOUaeiou"
+end
+
+no_vowels(album1)
+no_vowels(album2)
+no_vowels(album3)
+
```

Comment:
Looks good. It is a good idea to have the `p` on the method invocation rather than the inner working of the method.